config/pipeline.yaml

```diff
@@ -384,6 +393,12 @@ scheduler:
   - job: kbuild-gcc-10-arm64
     <<: *build-k8s-all
 
+  - job: kbuild-gcc-10-arm64-dtbscheck
+    <<: *build-k8s-all
+    rules:
+      tree: next
+      branch: 'master'
```

Comment:
Those should go into the job definition, not the scheduler entry (this could be a future TODO though).